src/wp-includes/class-wp-theme-json.php

```diff
@@ -456,7 +456,7 @@ class WP_Theme_JSON {
 	public static function get_element_class_name( $element ) {
 		$class_name = '';
 
-		if ( array_key_exists( $element, static::__EXPERIMENTAL_ELEMENT_CLASS_NAMES ) ) {
+		if ( isset( static::__EXPERIMENTAL_ELEMENT_CLASS_NAMES[ $element ] ) ) {
```

Comment:
> Fatal error: Cannot use isset() on the result of an expression (you can use "null !== expression" instead) in ./src/wp-includes/class-wp-theme-json.php on line 459

Reply:
☝🏻 This is your PHP 5.6 error ;-)

Reply:
https://github.com/WordPress/gutenberg/pull/42567

Reply:
Just as a note for the record: running `isset()`on plain constants/class constants is fine, it's only array access on those constants which is problematic.

The ability to declare a `const` as an array was only added in PHP 5.6, the ability to do the same using `define()` was added in PHP 7.0, which is probably why the ability to use `isset()` on array access on constants was added in PHP 7.0.

Reply:
Perfect, thank you! I reverted the lines with the 5.6 issue, and added notes so we know what to do in the future after WP drops support for PHP 5.6 😄